packages/vega-util/src/mergeDeep.js

```diff
@@ -0,0 +1,33 @@
+import isArray from "./isArray";
+import isObject from "./isObject";
+import hasOwnProperty from "./hasOwnProperty";
+
+export default function mergeDeep(dest, ...src) {
+  for (const s of src) {
+    dest = mergeDeep_(dest, s);
+  }
+  return dest;
+}
+
+function mergeDeep_(dest, src) {
+  if (!isObject(src) || src === null) {
```

Comment:
Why are there strict equality checks for null? The vega-util `isObject(null)` returns `false`, so I think we can remove the extra check. Also on lines 24 and 26.